fe/fe-core/src/main/java/com/starrocks/sql/plan/ScalarOperatorToExpr.java

```diff
@@ -66,7 +68,15 @@
 
 public class ScalarOperatorToExpr {
     public static Expr buildExecExpression(ScalarOperator expression, FormatterContext descTbl) {
-        return expression.accept(new Formatter(), descTbl);
+        return expression.accept(new Formatter(ScalarOperatorToExpr::buildExecExpression), descTbl);
+    }
+
+    public static Expr buildExprIgnoreSlot(ScalarOperator expression, FormatterContext descTbl) {
+        return expression.accept(new IgnoreSlotFormatter(ScalarOperatorToExpr::buildExprIgnoreSlot), descTbl);
+    }
+
+    interface BuildExpr {
+        Expr build(ScalarOperator expression, FormatterContext descTbl);
```

Comment:
I didn't find the implement of this interface 👀

Reply:
new Formatter(ScalarOperatorToExpr::buildExecExpression),

new IgnoreSlotFormatter(ScalarOperatorToExpr::buildExprIgnoreSlot),